deploy/base/role_binding.yaml

```diff
@@ -3,7 +3,6 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
   name: security-profiles-operator
-  namespace: security-profiles-operator
```

Comment:
I think we should keep the `namespace` here, or this will break for non helm deployments.

Reply:
all tests are passing so probably this isn't an issue.

Reply:
I think it is correct in this way, these are non namespaced resources, you can check it on docs or with the command 
`kubectl api-resources --namespaced=true`